library/README.md

```diff
@@ -93,6 +93,23 @@ Fields:
 
 The criteria to create subscription to a light node in JSON Format:
 
+```ts
+{
+    contentTopics: string[];
+    pubsubTopic: string;
+}
+```
+
+Fields:
+
+- `contentTopics`: Array of content topics.
+- `topic`: pubsub topic.
+
+
+### `LegacyFilterSubscription` type
+
+The criteria to create subscription to a light node in JSON Format:
```

Comment:
Confusing wording: The create subscription would be from a light node not to it. Wherever we are referring to creation of subscription, this can be modified to avoid confusion.